### ISSUE TITLE
The finetuning code for the SA-CLIP

### ISSUE THREAD

**Issue by c-pupil** [jiangyitong/AutoDIR#15]
Dear author:
First of all, thank you for your excellent work. Additionally, I would like to ask when you plan to release the code for the train CLIP part. Wishing you all the best in your work!

**Comment by jiangyitong** [comment]
Yes, thanks for asking. The training code will be released soon.

**Comment by Youqiang-Gui** [comment]
> Yes, thanks for asking. The training code will be released soon.
Hello, this is an excellent work. I would like to ask when the code of CLIP fine-tuning can be made public. Thank you for your trouble.
